AdyenDropIn/Models/PaymentMethodsConfiguration.swift

```diff
@@ -39,41 +36,53 @@ public extension DropInComponent {
         public init(clientKey: String) {
             self.clientKey = clientKey
         }
-        
-        /// Card component related configuration.
-        public final class CardConfiguration {
-            
-            /// Indicates if the field for entering the holder name should be displayed in the form. Defaults to false.
-            public var showsHolderNameField = false
-            
-            /// Indicates if the field for storing the card payment method should be displayed in the form. Defaults to true.
-            public var showsStorePaymentMethodField = true
-            
-            /// Indicates whether to show the security code field in the form.
-            public var showsSecurityCodeField = true
-
-            /// Stored card configuration.
-            public var stored = StoredCardConfiguration()
-            
-        }
+    }
+
+    /// Card component related configuration.
+    final class CardConfiguration {
```

Comment:
this should be public, right?

Reply:
It is public because extension is public 🤔
I can change that..